src/components/sidebar.tsx

```
@@ -90,7 +53,7 @@ export default function Sidebar({ sidebarOpen, setSidebarOpen, links }: SidebarP
                 <Store className="w-6 h-6" />
                 <span>STORE</span>
               </Link>
-              <button onClick={() => setSidebarOpen(false)} className="md:hidden">
+              <button onClick={() => setSidebarOpen(false)} className="md:hidden text-2xl">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)